website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md

```diff
@@ -0,0 +1,145 @@
+
+Let’s generate a [Databricks personal access token (PAT)](https://docs.databricks.com/sql/user/security/personal-access-tokens.html) for Development:
+
+1. In Databricks, click on your Databricks username in the top bar and select User Settings in the drop down.
+2. On the Access token tab, click Generate new token.
+3. Click Generate.
+4. Copy the displayed token and click Done. (don’t lose it!)
+
+
+For your development credentials/profiles.yml:
+
+1. Set your default catalog to dev.
+2. Your developer schema should be named after yourself. We recommend dbt_<first_name_initial><last_name>.
+
+By doing this, dbt will create the developer schemas if it doesn’t already exist in the **dev** catalog but can be overridden to specify sources or production targets.
+
+### Defining your dbt deployment environment
+
+Last, we need to give dbt a way to deploy code outside of development environments. To do so, we’ll use dbt [environments](https://docs.getdbt.com/docs/collaborate/environments) to define the production targets that end users will interact with.
+
+Core projects can use [targets in profiles](https://docs.getdbt.com/docs/get-started/connection-profiles#understanding-targets-in-profiles) to separate environments. [dbt Cloud environments](https://docs.getdbt.com/docs/get-started/develop-in-the-cloud#set-up-and-access-the-cloud-ide) allow you to define environments via the UI and [schedule jobs](https://docs.getdbt.com/docs/get-started/getting-started/building-your-first-project/schedule-a-job) for specific environments.
+
+Let’s set up our deployment environment:
+
+1. In Databricks, [set up your service principal’s token](https://docs.databricks.com/dev-tools/service-principals.html#use-curl-or-postman). To do this, you will need to open up your terminal.
+2. In your terminal, define 2 environment variables:
+    1. **DATABRICKS_HOST** - the url used to log into your Databricks workspace
+    2. **DATABRICKS_TOKEN** - your [personal access token](https://docs.databricks.com/dev-tools/auth.html#pat) (also used for dbt Cloud developer credentials)
+
+```bash
+export DATABRICKS_HOST="https://dbc-a1b2345c-d6e78.cloud.databricks.com"
+export DATABRICKS_TOKEN="dapi1234567890b2cd34ef5a67bc8de90fa12b"
+```
+
+3. Next, run a curl command to get the “application_id” of your service principal. You can also get this value from the Databricks workspace admin console > Service Principals > Edit
+
+```bash
+curl -X GET \
+${DATABRICKS_HOST}/api/2.0/preview/scim/v2/ServicePrincipals \
+--header "Authorization: Bearer ${DATABRICKS_TOKEN}" \
+| jq .
+```
+
+4. Then you create a JSON file with the application ID retrieved from the above command, a comment to describe what the token will be used for, and the amount of time the token will be valid (in seconds) before it expires and will need to be regenerated.
+
+```bash
+vim create-service-principal.json
```

Comment:
Likewise, let's not demand people use vim. We should just tell them to create a text file, and they can do it however they see fit (but we'll need to make clear that it has to be in the current working directory)